int32 states
string movementName
int32 signalGroup

StateTimeSpeed StateTimeSpeed
ManeuverAssistList ManeuverAssistList


